go/kbnm/hostmanifest/whitelist.go

```diff
@@ -0,0 +1,61 @@
+package hostmanifest
+
+import (
+	"encoding/json"
+	"os"
+	"os/user"
+	"path/filepath"
+)
+
+// whitelistPath is used for installing the whitelist as a JSON into a given
+// path on disk.
+type whitelistPath struct {
+	// Root is the path of the NativeMessage whitelists for root user.
```

Comment:
"For the root user" is a little confusing here. It applies to all users, right? Just lives in a system path that can only be written by root?